seerep_srv/seerep_core_fb/src/core_fb_conversion.cpp

```diff
@@ -125,6 +125,164 @@ seerep_core_msgs::QueryTf CoreFbConversion::fromFb(const seerep::fb::TransformSt
   return queryTf;
 }
 
+seerep_core_msgs::camera_intrinsics CoreFbConversion::fromFb(const seerep::fb::CameraIntrinsics& ci)
+{
+  seerep_core_msgs::camera_intrinsics ciCore;
+
+  CoreFbConversion::fromFbDataHeader(ci.header(), ciCore.header, seerep_core_msgs::Datatype::Unknown);
+
+  ciCore.height = ci.height();
+  ciCore.width = ci.width();
+  ciCore.distortion_model = ci.distortion_model()->str();
+
+  // traverse distortion list and convert
+  if (flatbuffers::IsFieldPresent(&ci, seerep::fb::CameraIntrinsics::VT_DISTORTION))
```

Comment:
something isn't working here. The values assigned to the vectors in ciCore do not match the values defined in gRPC_fb_addCameraIntrinsics.py